services/app-web/src/components/DatePicker/Day.stories.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+
+import { Day } from './Day'
+
+/*
+// THIS STORY FOR INTERNAL DEVELOPMENT ONLY
```

Comment:
Can you also drop a README in this DatePicker component folder about why we are doing this, when, and under what condition we could go back to using react-uswds version?